### PR TITLE
databases/week3

### PR DESCRIPTION
**Pull request by semih1239**

### CODE REVIEW

File: databases/week3/hyf_week3_homework.sql [hunk 1]
```diff
@@ -0,0 +1,165 @@
+SET
+  NAMES utf8mb4;
```
Comment (sboutzen) [inline databases/week3/hyf_week3_homework.sql]: No need for a new line here

File: databases/week3/hyf_week3_homework.sql [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+-- Get reservations for a specific meal sorted by created_date
+
+SELECT *
+FROM reservation
+JOIN meal ON meal.id = reservation.meal_id
+WHERE meal.title LIKE '%yaprak%'
+ORDER BY reservation.created_date ASC;
+
+-- Sort all meals by average number of stars in the reviews
```
Comment (sboutzen) [inline databases/week3/hyf_week3_homework.sql:159]: Nitpicking, but put your comment directly above the query it describes, it gives a stronger visual cue as to where it belongs

**Review by sboutzen — COMMENTED** [review]
Well done, small comments :)

File: databases/week3/hyf_week3_homework.sql [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+SELECT meal.* , review.stars
```
Comment (sboutzen) [inline databases/week3/hyf_week3_homework.sql]: This does not give you an average of the stars. You can use the AVG function, described here: https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_avg, and group by meal id and order by avg

Comment (semih1239) [reply databases/week3/hyf_week3_homework.sql]: You are right. I will update my codes 👍

**Review by semih1239 — COMMENTED** [review]
> Well done, small comments :)

Thanks for your feedbacks. I will update my codes

**Pull request merged**